extensions/notebook/src/extension.ts

```diff
@@ -85,9 +85,6 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 		return dialog.createDialog();
 	}));
 
-	extensionContext.subscriptions.push(vscode.commands.registerCommand('_notebook.command.new', async (options?: azdata.nb.NotebookShowOptions) => {
```

Comment:
Did you remove all the references to this command, it's still referred here https://github.com/microsoft/azuredatastudio/blob/18f090b8c273d44320d223ecbd29632eb3d79efd/extensions/notebook/package.json#L102